procrastinate/builtin_tasks.py

```diff
@@ -17,7 +17,7 @@ def remove_old_jobs(
 
 # Register your builtin tasks here
 def register_builtin_tasks(app) -> None:
-    app.task(
+    app.builtin_tasks["procrastinate.builtin_tasks.remove_old_jobs"] = app.task(
```

Comment:
do we really need to reiterate `procrastinate.builtin_tasks` ?

Reply:
I changed my mind the commit after :)